.github/workflows/ci-workflow.yml

```diff
@@ -147,12 +152,17 @@ jobs:
           java-version: '11'
       - name: Maven Build
         env:
+          NIFI_CI_LOCALE: >-
```

Comment:
This environment variable will normally not be set, so the new unit test will be deactivated.